planetmint/lib.py

```diff
@@ -143,23 +143,28 @@ def store_bulk_transactions(self, transactions):
         assets = []
         txn_metadatas = []
 
-        for t in transactions:
-            transaction = t.tx_dict if t.tx_dict else rapidjson.loads(rapidjson.dumps(t.to_dict()))
+        for tx in transactions:
+            transaction = tx.tx_dict if tx.tx_dict else rapidjson.loads(rapidjson.dumps(tx.to_dict()))
+
+            try:
+                tx_assets = transaction.pop("assets")
+            except KeyError:
+                tx_assets = transaction.pop("asset")
```

Comment:
how about creating a function that checks what key is there and returns the key to pop from the dict
def get_supported_asset_key(transactions: dict) -> str:
    if "asset" in transactions:
        return "asset"
    
    return "assets"`

`asset_key = get_supported_asset_key(transaction)
tx_assets = transaction.pop(asset_key)`

Reply:
good idea, this function exists already. will use it!